### DISCUSSION TITLE
Is there a way to access the values of the Rate Limit metrics?

### DISCUSSION THREAD

**Discussion by naotee** [sidekiq/sidekiq#5349] [Q&A] [answered]
I am using Sidekiq::Limiter to control the number of concurrent executions of a job. 
When the number of concurrent executions is exceeded, I would like to give the user an on-screen message saying that the job is "busy".
So I would like to get the value in the attached image, is there any way to get it?
<img width="792" alt="スクリーンショット 2022-05-30 17 55 50" src="https://user-images.githubusercontent.com/13898347/170959010-480cfe80-a9aa-4be9-963b-5375452168a6.png">

**Comment by mperham** [comment, marked as answer]
Everything you see in the web ui is available via an API, you just have to find it. Open the gem and start browsing.

**Reply by mperham** [reply]
You’re looking for sidekiq-ent/limiter/status.rb

**Reply by naotee** [reply]
Thank you very much. I got what I wanted.
